spec/devise/passwordless/login_token_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'timecop'
+
+RSpec.describe Devise::Passwordless::LoginToken do
+  describe 'encryption and decryption' do
+    let(:user) { double(:user, email: 'email@example.com', to_key: 12345) }
+
+    before do
+      Timecop.freeze(Time.new(2023,1,1))
```

Comment:
In order to avoid some bad surprises from differing time zones, I think it would be better to use `Time.utc` here:

```suggestion
      Timecop.freeze(Time.utc(2023, 1, 1))
```